src/cm/action.rs

```diff
@@ -66,6 +67,7 @@ pub const NAMES: [&str; LEN] = [
     "jump_to_end",
     "next_search_match",
     "prev_search_match",
+    "edit_shell",
```

Comment:
What's the point of this action if we edit shells using shell list

Reply:
True, it is removed now.